### PR TITLE
week6 homework

### PR DESCRIPTION
**Pull request by dinythomas89**

### CODE REVIEW

File: week6/Products.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    public ProductRepository(IConfiguration configuration)
+    {
+        this.connectionString = configuration.GetConnectionString("Default");
+    }
+
+    public async Task<IEnumerable<Product>> GetProducts()
+    {
+        using var connection = new MySqlConnection(connectionString);
+        var products = await connection.QueryAsync<Product>("SELECT * FROM dotnetweek6.products");
+        return (IEnumerable<Product>)products;
```
Comment (tpetrina) [inline week6/Products.cs]: What happens if you just write `return products`?

File: week6/Products.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        await using var connection = new MySqlConnection(connectionString);
+        var updatedUser = await connection.ExecuteAsync($"UPDATE products SET name=@name, price=@price, description=@description WHERE id={id}", product);
+        return product;
+    }
+
+    public async Task<Product> DeleteProduct(int id)
+    {
+        await using var connection = new MySqlConnection(connectionString);
+        var deletedUser = await connection.ExecuteAsync($"DELETE FROM products WHERE id={id}");
+        return (Product)Results.Ok();
```
Comment (tpetrina) [inline week6/Products.cs]: This line should not work.
Deleting should usually return nothing - so change the return value to just `Task`

File: week6/Users.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        await using var connection = new MySqlConnection(connectionString);
+        var updatedUser = await connection.ExecuteAsync($"UPDATE user SET name=@name, email=@email WHERE id={id}", user);
+        return user;
+    }
+
+    public async Task<User> DeleteUser(int id)
+    {
+        await using var connection = new MySqlConnection(connectionString);
+        var deletedUser = await connection.ExecuteAsync($"DELETE FROM user WHERE id={id}", new { ID = id });
+        return (User)Results.Ok();
```
Comment (tpetrina) [inline week6/Users.cs]: This line should not work.
Deleting should usually return nothing - so change the return value to just `Task`

**Review by tpetrina — CHANGES_REQUESTED** [review]
Looks good, just small nitpick.
Try running all endpoints to see if they work the way you expect.

File: week6/Users.cs (hunk 3 above)
Comment (dinythomas89) [reply week6/Users.cs]: Thank you Toni for the valuable feedbacks

**Pull request merged**